RELEASE_NOTES.md

```diff
@@ -23,12 +25,15 @@
 - The `podman rename` command has been improved to be more atomic, eliminating many race conditions that could potentially render a renamed container unusable.
 - Detection of which OCI runtimes run using virtual machines and thus require custom SELinux labelling has been improved ([#9582](https://github.com/containers/podman/issues/9582)).
 - The hidden `--trace` option to `podman` has been turned into a no-op. It was used in very early versions for performance tracing, but has not been supported for some time.
+- The `podman generate systemd` command now generates `RequiresMountsFor` lines to ensure necessary storage directories are mounted before systemd starts Podman.
+- Podman will now emit a warning when `--tty` and `--interactive` are both passed, but `STDIN` is not a TTY. This will be made into an error in the next major Podman release some time next year.
```

Comment:
Timing for this might be off on different distros.  I'd just drop the next year bit and just stick with the next release.
```suggestion
- Podman will now emit a warning when `--tty` and `--interactive` are both passed, but `STDIN` is not a TTY. This will be made into an error in the next major Podman release.
```

Reply:
These are upstream changelogs, I'm not really worried about distro timeframes for these.